
I am currently a Research Associate in the Data Science Institute at Imperial College London.


## Employment

* September 2018- **Research Associate** in the [Data Science Institute](https://www.imperial.ac.uk/data-science/) at Imperial College London. The institute has a 64-screen tiled display with a 130 megapixel resolution driven by either 5 or 32 computers. I am part of the team that develops the software that controls this environment, including the [OVE framework](https://ove.readthedocs.io/en/stable/), and I also create visualizations in a range of domains. Both activities involve considerable amounts of programming using Python and JavaScript.


## Education

*  2014-2019: **D.Phil student** on the 4-year EPSRC/BBSRC [Synthetic Biology Centre for Doctoral Training](http://www.synbio-cdt.ac.uk/) program at the University of Oxford, supervised by **Prof. Antonis Papachristodoulou** (Department of Engineering Science, University of Oxford). My research focused on design tools for synthetic biology, and in particular the visualisation and human-computer interaction problems that must be addressed for these to be effective.

* **BA and M.Eng (Cantab)** conferred 2014

* 2013-14: Part IIB of the Engineering Tripos (Merit). Completed project on
Computational Auditory Scene Analysis, supervised by Dr. Rich Turner. Studied the modules: 

    Robust and non-linear systems and control, Optimal and predictive control, Practical optimisation, Statistical Pattern Processing, Machine Learning, Computational neuroscience, Molecular modelling, Accounting and finance.


* 2012-12: Part IIA of the Engineering Tripos (Class I). Studied the modules:

    Signals and systems, Systems and Control, Computer and Network Systems, Software Engineering and Design, Medical Imaging and 3D Computer Graphics,  Mathematical Physiology, Introduction to Neuroscience, Introduction to Molecular Bioengineering, Biomaterials, Operations Management for Engineers


* 2011-12: Part I of the Natural Sciences Tripos (Class II). Studied Biology of Cells, Physiology of Organisms, Chemistry, and Mathematics B at Part IA. Studied Biochemistry & Molecular Biology, Cell & Developmental Biology, and Mathematics at Part IB.

* A-levels in Physics, Chemistry, Biology, Mathematics and Further Mathematics.

## Research Outputs

### Journal articles

* Senaka Fernando, **James Scott-Brown**, Ovidiu Serban, David Birch, David Akroyd, Miguel Molina-Solana, Thomas Heinis, Yike Guo. **OVE: A web framework for scalable rendering of data visualizations**. *Future Generation Computer Systems* (under review)

* Jacob Beal, Tramy Nguyen, Thomas E. Gorochowski, Angel Goni-Moreno, **James Scott-Brown**, James Alastair McLaughlin, Curtis Madsen, Benjamin Aleritsch, Bryan Bartley, Shyam Bhakta, Mike Bissell, Sebastian Castillo Hair, Kevin Clancy, Augustin Luna, Nicolas Le Novere, Zach Palchick, Matthew Pocock, Herbert Sauro, John T. Sexton, Jeffrey J. Tabor, Christopher A. Voigt, Zach Zundel, Chris Myers, Anil Wipat. **Communicating Structure and Function in Synthetic Biology Diagrams.** *ACS Synthetic Biology*. July 2019. DOI: [10.1021/acssynbio.9b00139](http://dx.doi.org/10.1021/acssynbio.9b00139)

* Curtis Madsen, Angel Goni Moreno, Zachary Palchick, Umesh P, Nicholas Roehner, Bryan Bartley, Swapnil Bhatia, Shyam Bhakta, Mike Bissell, Kevin Clancy, Robert Sidney Cox, Thomas Gorochowski, Raik Grunberg, Augustin Luna, James McLaughlin, Tramy Nguyen, Nicolas Le Novere, Matthew Pocock, Herbert Sauro, **James Scott-Brown**, John T. Sexton, Guy-Bart Stan, Jeffrey J. Tabor, Christopher A. Voigt, Zach Zundel, Chris Myers, Jacob Beal, Anil Wipat. **Synthetic Biology Open Language Visual (SBOL Visual) Version 2.1.** *Journal of Integrative Bioinformatics*. June 2019. DOI: [10.1515/jib-2018-0101](https://doi.org/10.1515/jib-2018-0101)

* Curtis Madsen, Angel Goni Moreno, Umesh P, Zachary Palchick, Nicholas Roehner, Christian Atallah, Bryan Bartley, Kiri Choi, Robert Sidney Cox, Thomas Gorochowski, Raik Gr√ºnberg, Chris Macklin, James McLaughlin, Xianwei Meng, Tramy Nguyen, Matthew Pocock, Meher Samineni, **James Scott-Brown**, Ysis Tarter, Michael Zhang, Zhen Zhang, Zach Zundel, Jacob Beal, Michael Bissell, Kevin Clancy, John H. Gennari, Goksel Misirli, Chris Myers, Ernst Oberortner, Herbert Sauro, Anil Wipat. **Synthetic Biology Open Language (SBOL) Version 2.3.** *Journal of Integrative Bioinformatics*. June 2019. DOI: [10.1515/jib-2019-0025](https://doi.org/10.1515/jib-2019-0025)

*  Melis Kayikci, AJ Venkatakrishnan, **James Scott-Brown**, Charles Ravarani, Mr. Tilman Flock, M. Madan Babu. **Visualization and analysis of non-covalent contacts in biomolecules.** *Nature Structural & Molecular Biology*. Jan 2018. DOI: [10.1038/s41594-017-0019-z](http://dx.doi.org/10.1038/s41594-017-0019-z)

* **James Scott-Brown**, Antonis Papachristodoulou. **sbml-diff: A tool for visually comparing SBML models in synthetic biology.**  *ACS Synthetic Biology*. Dec 2016. DOI: [10.1021/acssynbio.6b00273](http://dx.doi.org/10.1021/acssynbio.6b00273)

* **James Scott-Brown**, O A Cunningham, B C Goad. **How hot can a fire piston get?**, *Physics Education* 45, 2010. DOI:[10.1088/0031-9120/45/4/F04](http://dx.doi.org/10.1088/0031-9120/45/4/F04)

### Pre-prints

*  **James Scott-Brown**, Antonis Papachristodoulou. **Visual representation of experimental protocols.** bioRxiv DOI: [10.1101/226852](https://doi.org/10.1101/226852))


### Conference Talks

* August 2017: orally presented an abstract about interfaces for expressing laboratory protocols at the 9th International Workshop on Bio-Design Automation (IWBDA) in Pittsburgh, USA.

* August 2016: orally presented an abstract about visually presenting and comparing SBML models at the 8th International Workshop on Bio-Design Automation (IWBDA) in Newcastle, UK. 

### Conference Posters

* **James Scott-Brown**. *A Radial View of the BioVis Challenge 2019 Dataset*. BioVis Workshop (at IEEE VIS). October 2019.
* **James Scott-Brown**, Antonis Papachristodoulou. *Visualization of Temporal Logic Specifications*. Eurovis, June 2017. DOI: [10.2312/eurp.20171183](http://dx.doi.org/10.2312/eurp.20171183)
* **James Scott-Brown**, Antonis Papachristodoulou. *Visual Comparison of SBML Models*. Computational Modeling in Biology (COMBINE), September 2016 

### Conference position papers

* **James Scott-Brown** *Presenting Visualization Guideline Collections*, at VisGuides:
2nd Workshop on the Creation, Curation, Critique and Conditioning of Principles and Guidelines in Visualization (a workshop at IEEE VIS in Berlin), October 2018. [PDF](https://c4pgv.dbvis.de/Scott-Brown_2018.pdf)


## Professional service
* I maintain [visperception.com](http://visperception.com), a bibliography of experimental studies of the perception of visualizations.
* I aggregated links to open-access preprints and materials for EuroVis papers in [2018](http://oa-eurovis.jamesscottbrown.com/2018) and [2019](http://oa-eurovis.jamesscottbrown.com/2019/).

## Projects

* 2014: **DTC mini-project 1**, on 'External Control of Gene Expression', supervised by **Prof. Mario di Bernardo** (Department of Engineeirng Mathematics, University of Bristol). This involved designing and simulating controllers that would be implemented in software and control living cells in a microfluidic device. 

* 2014: **DTC mini-project 2**, on 'Applications of cell-cell communication to Synthetic Biology', supervised by **Prof. Antonis Papachristodoulou** (Department of Engineering Science, University of Oxford). This involved modelling novel synthetic circuits that exploited quorum sensing systems.

* 2013: Software Developer Intern at **The MathWorks**. Prototyped a web GUI using JavaScript and the Dojo toolkit. Also implemented a new graph layout and improved GUI for a dependency graph viewer.

* 2012: Student placement with **Dr Madan Babu Mohan's** group at the MRC Laboratory of Molecular Biology. Constructed a website to visualise and compare networks of non-covalent interactions within proteins. I implemented the front-end in JavaScript with D3.js, and the backend in C++. This project resulted in a paper in *Nature Structural & Molecular Biology*.

* 2011: Student placement with **Dr Bill Schafer's** group at the MRC Laboratory of Molecular Biology. Wrote a MATLAB program to play videos of experiments, and interact with metadata and annotations stored in both a MySQL database and binary files. I also had some exposure to *C. elegans* experimental methods, including behavioural assays and calcium imaging.

* 2010: Student placement with **Prof. Jeremy Frey's** group at the University of Southampton. This was mostly spent writing a Ruby on Rails web application to collect experimental data.

* 2008: Student placement with **Prof. Jeremy Frey's** group. This included writing Perl scripts as part of a laboratory automation project.

