
I am currently a Software Engineer in the Big Data Institute at the University of Oxford. I am also working part-time as a Research Associate at the University of Edinburgh on a short-term contract.


## Employment

* April 2020-: **Research Associate** in the [Visual+Inteactive Data](https://visualinteractivedata.github.io/bach.html) group at the University of Edinburgh (60% FTE).
My main responsibility in this role is to create a tool to help support the writing of local clinical guidelines by enabling clinicians to find similar existing guidelines that have been shared.
This is intended to help the translation of research findings about COVID-19 into clinical practice.
I am also involved in various smaller projects related to visualisation in support of [ISARIC 4C](http://isaric4c.net/) (the Coronavirus Clinical Characterisation Consortium).
Delivering these projects involves designing new visualisations and user interfaces, and implementing them using Python and JavaScript.

* January 2020-: **Software Engineer** in the [Centre for Population Approaches for Non-Communicable Disease Prevention](https://www.ndph.ox.ac.uk/cpnp) within the Nuffield Department of Population Health/[Big Data Institute](https://www.bdi.ox.ac.uk/) at the University of Oxford. I work on a tool that scrapes the websites of online retailers in order to construct a longitudinal dataset of the prices and nutritional content of foods ([FoodDB](https://www.ndph.ox.ac.uk/cpnp/research-projects/fooddb-and-myshop)). I am also involved in research using this dataset, including the creation of a browser extension to help consumers make healthier food choices by providing information at the point-of-purchase (BetterBasket).

* September 2018-January 2020: **Research Associate** in the [Data Science Institute](https://www.imperial.ac.uk/data-science/) at Imperial College London. The institute has a 64-screen tiled display with a 130 megapixel resolution driven by either 5 or 32 computers. I was part of the team that develops the software that controls this environment, including the [OVE framework](https://ove.readthedocs.io/en/stable/), and I also created visualizations in a range of domains. Both activities involved considerable amounts of programming using Python and JavaScript.


## Education

*  2014-2019: **D.Phil student** on the 4-year EPSRC/BBSRC [Synthetic Biology Centre for Doctoral Training](http://www.synbio-cdt.ac.uk/) program at the University of Oxford, supervised by **Prof. Antonis Papachristodoulou** (Department of Engineering Science, University of Oxford). My research focused on design tools for synthetic biology, and in particular the visualisation and human-computer interaction problems that must be addressed for these to be effective.

* **BA and M.Eng (Cantab)** conferred 2014. I read Part IA and IB (first and second year) of the Natural Sciences Tripos, and then Part IIA and IIB (third and fourth year) of the Engineering Tripos.

* A-levels in Physics, Chemistry, Biology, Mathematics and Further Mathematics.

## Research Outputs

### Journal articles

1. Benjamin Hatch, Linhao Meng, Jeanet Mante, James A. McLaughlin, **James Scott-Brown**, Chris J. Myers. **VisBOL 2 - Improving Web-based Visualization for Synthetic Biology Designs**. *Under review (submitted 7 April 2021)*.

1. **Development and validation of the ISARIC 4C Deterioration model for adults hospitalised with COVID-19: a prospective cohort study**. *Lancet Respiratory Medicine*. April 2021. DOI: [10.1016/S2213-2600(20)30559-2](https://dx.doi.org/10.1016/S2213-2600(20)30559-2) [This was a consortium paper with a large number of authors - my contribution was an interactive tool to calculate the Deterioration score for a patient, and visualize this is comparison to the full patient cohort.]

1. James Alastair McLaughlin, Jacob Beal, Goksel Misirli, Raik Gr√ºnberg, Bryan A Bartley, **James Scott-Brown**, Prashant Vaidyanathan, Pedro Fontanarrosa, Ernst Oberortner, Anil Wipat, Thomas Edward Gorochowski, Chris John Myers. **The Synthetic Biology Open Language (SBOL) Version 3: Simplified Data Exchange for Bioengineering**. *Frontiers in Bioengineering and Biotechnology*. September 2020.DOI: [10.3389/fbioe.2020.01009](https://dx.doi.org/10.3389/fbioe.2020.01009)

1. Senaka Fernando, **James Scott-Brown**, Ovidiu Serban, David Birch, David Akroyd, Miguel Molina-Solana, Thomas Heinis, Yike Guo. **OVE: A web framework for scalable rendering of data visualizations**. *Future Generation Computer Systems*. Published online 11 June 2020. DOI: [10.1016/j.future.2020.06.011](https://doi.org/10.1016/j.future.2020.06.011)

1. Jacob Beal, Tramy Nguyen, Thomas E. Gorochowski, Angel Goni-Moreno, **James Scott-Brown**, James Alastair McLaughlin, Curtis Madsen, Benjamin Aleritsch, Bryan Bartley, Shyam Bhakta, Mike Bissell, Sebastian Castillo Hair, Kevin Clancy, Augustin Luna, Nicolas Le Novere, Zach Palchick, Matthew Pocock, Herbert Sauro, John T. Sexton, Jeffrey J. Tabor, Christopher A. Voigt, Zach Zundel, Chris Myers, Anil Wipat. **Communicating Structure and Function in Synthetic Biology Diagrams.** *ACS Synthetic Biology*. July 2019. DOI: [10.1021/acssynbio.9b00139](http://dx.doi.org/10.1021/acssynbio.9b00139)

1. Melis Kayikci, AJ Venkatakrishnan, **James Scott-Brown**, Charles Ravarani, Tilman Flock, M. Madan Babu. **Visualization and analysis of non-covalent contacts in biomolecules.** *Nature Structural & Molecular Biology*. Jan 2018. DOI: [10.1038/s41594-017-0019-z](http://dx.doi.org/10.1038/s41594-017-0019-z)

1. **James Scott-Brown**, Antonis Papachristodoulou. **sbml-diff: A tool for visually comparing SBML models in synthetic biology.**  *ACS Synthetic Biology*. Dec 2016. DOI: [10.1021/acssynbio.6b00273](http://dx.doi.org/10.1021/acssynbio.6b00273)

1. **James Scott-Brown**, Oscar Cunningham, Ben C Goad. **How hot can a fire piston get?**, *Physics Education* 45. July 2010. DOI:[10.1088/0031-9120/45/4/F04](http://dx.doi.org/10.1088/0031-9120/45/4/F04)


### Specifications published in journals

#### SBOL

1. Hasan Baig, Pedro Fontanarrosa, Vishwesh Kulkarni, James McLaughlin, Prashant Vaidyanathan, Bryan Bartley, Jacob Beal, Matthew Crowther ,Thomas Gorochowski, Raik Grunberg, Goksel Misirli, **James Scott-Brown**, Ernst Oberortner, Anil Wipat, Chris Myers. **Synthetic Biology Open Language (SBOL) Version 3.0.0**. *Journal of Integrative Bioinformatics*. June 2020. DOI: [10.1515/jib-2020-0017](https://dx.doi.org/10.1515/jib-2020-0017).

1. Curtis Madsen, Angel Goni Moreno, Umesh P, Zachary Palchick, Nicholas Roehner, Christian Atallah, Bryan Bartley, Kiri Choi, Robert Sidney Cox, Thomas Gorochowski, Raik Grunberg, Chris Macklin, James McLaughlin, Xianwei Meng, Tramy Nguyen, Matthew Pocock, Meher Samineni, **James Scott-Brown**, Ysis Tarter, Michael Zhang, Zhen Zhang, Zach Zundel, Jacob Beal, Michael Bissell, Kevin Clancy, John H. Gennari, Goksel Misirli, Chris Myers, Ernst Oberortner, Herbert Sauro, Anil Wipat. **Synthetic Biology Open Language (SBOL) Version 2.3.** *Journal of Integrative Bioinformatics*. June 2019. DOI: [10.1515/jib-2019-0025](https://doi.org/10.1515/jib-2019-0025)


#### SBOL Visual
1. Hasan Baig, Pedro Fontanarossa, Vishwesh Kulkarni, James McLaughlin, Prasant Vaidyanathan, Bryan Bartley, Swapnil Bhatia, Shyam Bhakta, Mike Bissell, KevinClancy, Robert Sidney Cox, Angel Goni Moreno, Thomas Gorochowski, Raik Grunberg, Augustin Luna, Curtis Madsen, Goksel Misirli, Tramy Nguyen, Nicolas Le Novere, Zachary Palchick, Matthew Pocock, Nicholas Roehner, Herbert Sauro, **James Scott-Brown**, John T. Sexton, Guy-Bart Stan, Jeffrey J.Tabor, Marta Vazquez Vilar, Christopher A. Voigt, Anil Wipat, David Zong, Zach Zundel, Jacob Beal, Chris Myers. **Synthetic Biology Open Language Visual (SBOL Visual) Version 2.2.** *Journal of Integrative Bioinformatics*. 
Published online 10 June 2020. DOI: [10.1515/jib-2018-0001](https://doi.org/10.1515/jib-2018-0001)

1. Curtis Madsen, Angel Goni Moreno, Zachary Palchick, Umesh P, Nicholas Roehner, Bryan Bartley, Swapnil Bhatia, Shyam Bhakta, Mike Bissell, Kevin Clancy, Robert Sidney Cox, Thomas Gorochowski, Raik Grunberg, Augustin Luna, James McLaughlin, Tramy Nguyen, Nicolas Le Novere, Matthew Pocock, Herbert Sauro, **James Scott-Brown**, John T. Sexton, Guy-Bart Stan, Jeffrey J. Tabor, Christopher A. Voigt, Zach Zundel, Chris Myers, Jacob Beal, Anil Wipat. **Synthetic Biology Open Language Visual (SBOL Visual) Version 2.1.** *Journal of Integrative Bioinformatics*. June 2019. DOI: [10.1515/jib-2018-0101](https://doi.org/10.1515/jib-2018-0101)



### Pre-prints

1.  **James Scott-Brown**, Antonis Papachristodoulou. **Visual representation of experimental protocols.** bioRxiv DOI: [10.1101/226852](https://doi.org/10.1101/226852)


### Conference Talks

1. August 2017: presented an abstract about interfaces for expressing laboratory protocols at the 9th International Workshop on Bio-Design Automation (IWBDA) in Pittsburgh, USA.

1. August 2016: presented an abstract about visually presenting and comparing SBML models at the 8th International Workshop on Bio-Design Automation (IWBDA) in Newcastle, UK. 

### Conference Posters

1. **James Scott-Brown**. *A Radial View of the BioVis Challenge 2019 Dataset*. BioVis Workshop (at IEEE VIS). October 2019. ([PDF](http://blog.jamesscottbrown.com/biovis_contest_2019/poster.pdf))
1. **James Scott-Brown**, Antonis Papachristodoulou. *Visualization of Temporal Logic Specifications*. Eurovis, June 2017. DOI: [10.2312/eurp.20171183](http://dx.doi.org/10.2312/eurp.20171183)
1. **James Scott-Brown**, Antonis Papachristodoulou. *Visual Comparison of SBML Models*. Computational Modeling in Biology (COMBINE), September 2016.

### Conference position papers

* **James Scott-Brown** *Presenting Visualization Guideline Collections*, at VisGuides:
2nd Workshop on the Creation, Curation, Critique and Conditioning of Principles and Guidelines in Visualization (a workshop at IEEE VIS in Berlin), October 2018. ([PDF](https://c4pgv.dbvis.de/Scott-Brown_2018.pdf))


## Professional service

* I have been contrbuting to the [Synthetic Biology Open Language (SBOL)](https://sbolstandard.org/) and SBOL Visual standards for several years, as was elected as an Editor of the standards in 2021
* I have reviewed submissions to the Eurovis conference
* I maintain [visperception.com](http://visperception.com), a bibliography of experimental studies of the perception of visualizations.
* I aggregated links to open-access preprints and materials for EuroVis papers in [2018](http://oa-eurovis.jamesscottbrown.com/2018) and [2019](http://oa-eurovis.jamesscottbrown.com/2019/); this effort was then made largely redundant by a policy change that made all EuroVis papers open-access from 2020 onwards

## Projects

* 2014: **DTC mini-project 1**, on 'External Control of Gene Expression', supervised by **Prof. Mario di Bernardo** (Department of Engineeirng Mathematics, University of Bristol). This involved designing and simulating controllers that would be implemented in software and control living cells in a microfluidic device. 

* 2014: **DTC mini-project 2**, on 'Applications of cell-cell communication to Synthetic Biology', supervised by **Prof. Antonis Papachristodoulou** (Department of Engineering Science, University of Oxford). This involved modelling novel synthetic circuits that exploited quorum sensing systems.

* 2013: Software Developer Intern at **The MathWorks**. Prototyped a web GUI using JavaScript and the Dojo toolkit. Also implemented a new graph layout and improved GUI for a dependency graph viewer in Java.

* 2012: Student placement with **Dr Madan Babu Mohan's** group at the MRC Laboratory of Molecular Biology. Constructed a website to visualise and compare networks of non-covalent interactions within proteins. I implemented the front-end in JavaScript with D3.js, and the backend in C++. This project resulted in a paper in *Nature Structural & Molecular Biology*.

* 2011: Student placement with **Dr Bill Schafer's** group at the MRC Laboratory of Molecular Biology. Wrote a MATLAB program to play videos of experiments, and interact with metadata and annotations stored in both a MySQL database and binary files. I also had some exposure to *C. elegans* experimental methods, including behavioural assays and calcium imaging.

* 2010: Student placement with **Prof. Jeremy Frey's** group at the University of Southampton. This was mostly spent writing a Ruby on Rails web application to collect experimental data.

* 2008: Student placement with **Prof. Jeremy Frey's** group. This included writing Perl scripts as part of a laboratory automation project.
